smoke-tests/src/test/java/com/grafana/extensions/smoketest/SmokeTest.java

```diff
@@ -167,8 +167,8 @@ protected static int countSpansByAttributeValue(
             .count();
   }
 
-  protected static Stream<Span> getSpanStream(Collection<ExportTraceServiceRequest> traces) {
-    return traces.stream()
+  protected static Stream<Span> getSpanStream(Collection<ExportTraceServiceRequest> logs) {
```

Comment:
I can see why you change the fucntion below to metric, since is "return METRIC stream", but this function is "return SPAN stream", so why is not returning LOG stream?

Reply:
good point :smile: